app/assets/javascripts/markdown_previews.js

```diff
@@ -0,0 +1,28 @@
+function setup_all_markdown_writing_preview () {
```

Comment:
Identifier 'setup_all_markdown_writing_preview' is not in camel case.<br>'setup_all_markdown_writing_preview' is defined but never used.